setup.py

```diff
@@ -6,8 +6,9 @@
     description='FastSpring API integration for Flask',
     py_modules=['flask_fastspring'],
     install_requires=[
-        'Flask>=0.11',
+        'Flask>=3.0',
```

Comment:
Are the changes compatible with Flask 2.0? If so, use that version as the lower bound.